ide/coqide/FAQ.md

```diff
@@ -0,0 +1,81 @@
+# CoqIDE FAQ #
+
+### What is CoqIDE?
+
+A powerful GTK3-based graphical interface for Coq.
+See https://coq.inria.fr for more information.
+
+### How to enable Emacs keybindings?
+
+Insert the following line in your `gtkrc` file:
+```
+gtk-key-theme-name = "Emacs"
+```
+The location of this file is system-dependent. If you're running
+GNOME, you may use the graphical configuration tools.
+
+See also the µPG mode in CoqIDE preferences and help.
+
+### How to enable antialiased fonts?
+
+Set the `GDK_USE_XFT` variable to 1. This is the default with Gtk >= 2.2.
+If some of your fonts are not available, set `GDK_USE_XFT` to 0.
+
+### How to use those Forall (∀) and Exists (∃) pretty symbols?
+
+Thanks to the Notation features in Coq, you just need to insert these
+lines in your Coq Buffer :
+```
+Notation "∀ x : t, P" := (forall x:t, P) (at level 200, x ident).
+Notation "∃ x : t, P" := (exists x:t, P) (at level 200, x ident).
+```
+Copy-pasting of these lines from this file will not work outside of CoqIDE.
+You need to load a file containing these lines or to enter the `∀`
+using an input method (see next question below).
+To try it, just use `Require Import Utf8` from inside CoqIDE.
```

Comment:
The file `Utf8.v` has some junk that's been commented out since 2003.  It includes `Utf8_core.v` and provides these two definitions.  The rest is just comments.  These files can probably be merged.  Would you like to take on cleaning this up (in yet another PR)?  @herbelin, WDYT?

```
Notation "x ≤ y" := (le x y) (at level 70, no associativity).
Notation "x ≥ y" := (ge x y) (at level 70, no associativity).
```

Reply:
We are missing a way to incrementally add utf8 notations when files are required.

One issue is that different strategies are coexisting and we would need to agree on which ones we support:
- not relying on unicode characters saved in files, working instead at the UI level thanks to some emacs-like prettify;
- for each notation, propose an alternative utf8 variant which is activated by setting an option (e.g. one would set, in `Notations.v`, `Reserved Notation "x -> y" (at level 99, right associativity, y at level 200, utf8 "x → y").`, then, after `Set Utf8.`, the notation would be used for printing. For notations already using utf8 by default, e.g. `x ³`, we can propose the converse, that is a modifier of the form `ascii "x ^3"`.

In any cases, regarding `Utf8.v` and `Utf8_core.v`, they could be cleaned.